services/user.py

```diff
@@ -0,0 +1,58 @@
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> User:
+    user = {
+        "username": username,
+        "password": password,
+    }
+
+    if email is not None:

```

Comment:
Simplify it. No need to check those arguments for None.
Use like this:
```
if email:
   ...
```
and so on..